src/move/econia/sources/market.move

```diff
@@ -613,11 +603,11 @@ module econia::market {
         asks: AVLqueue<Order>,
         /// Bids AVL queue.
         bids: AVLqueue<Order>,
-        /// Cumulative number of maker orders placed on book.
+        /// Cumulative number of orders placed.
         counter: u64,
-        /// Event handle for maker events.
+        /// Deprecated field retained for backwards compatibility.
         maker_events: EventHandle<MakerEvent>,
-        /// Event handle for taker events.
+        /// Deprecated field retained for backwards compatibility.
         taker_events: EventHandle<TakerEvent>
```

Comment:
This doesn't really offer backwards compatibility. We should stop saying this, for reasons I've written [here](https://github.com/econia-labs/econia/pull/315#discussion_r1254266215).

Reply:
69f05ea modifies commentary per `Move.toml` nomenclature:

```toml
[package]
name = "Econia"
version = "4.1.0"             # Per SemVer.
upgrade_policy = "compatible"
```